### ISSUE TITLE
Fix handling of multiline strings

### ISSUE THREAD

**Issue by ad-si** [harelba/q#214]
example-accounts.csv:
```csv
email,sf-id,z-id,description
a@example.com,sf1,z1,"multiline
test
string 1"
b@example.com,sf2,z2,"multiline
test
string 2"
c@example.com,sf3,z3,"multiline
test
string 3"
```

example-tasks.csv:
```csv
value,relation-id
x,z1
y,z2
z,z3
```

The following command:

```sh
q --skip-header --output-header -d ',' \
'select * from example-accounts.csv \
join example-tasks.csv on `z-id` == `relation-id`
```

yields

```csv
email,s-id,z-id,description,value,relation-id
a@example.com,sf1,z1,multiline
test
string 1,x,z1
b@example.com,sf2,z2,multiline
test
string 2,y,z2
c@example.com,sf3,z3,multiline
test
string 3,z,z3
```

but must yield

```csv
email,s-id,z-id,description,value,relation-id
a@example.com,sf1,z1,"multiline
test
string 1",x,z1
b@example.com,sf2,z2,"multiline
test
string 2",y,z2
c@example.com,sf3,z3,"multiline
test
string 3",z,z3
```

Notice the additional quotes around the multiline string.

**Comment by harelba** [comment]
Hi, sorry for the too-late reply, crazy times.

The default output-wrapping mode of q is "minimal", which should have detected multiline-strings and quote them.

This wrapping mode should have detected the multiline string and quote it, and it doesn't. I'll try to see if this can be easily fixed.

In order to prevent the issue in the mean time, you can use `-W all` or `-W nonnumeric`, and the output will be re-readable by q and other csv tools.

Note that in terms of the actual data, q reads the multiline string properly:
```
$ q -H -d , "select count(*) from example-accounts.csv"
3
$ q -H -d , "select description from example-accounts.csv" -W all
"multiline
test
string 1"
"multiline
test
string 2"
"multiline
test
string 3"
```

**Comment by harelba** [comment]
yes, this is easily fixable. I'll integrate the fix into the next release

**Comment by harelba** [comment]
Fixed in version 2.0.19 which has just been released.
